### ISSUE TITLE
The Last Line of Defense

### ISSUE THREAD

**Issue by magamis** [dalaranwow/dalaran-wow#5400]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: The quest mobs are all evading the damage from the cannon.

**Expected behaviour**: Damage should be hitting the mob

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by magamis** [comment]
![wowscrnshot_041518_004751](https://user-images.githubusercontent.com/37825063/38769444-b25c6f02-4046-11e8-861c-ccad95ec754d.jpg)
![wowscrnshot_041518_004537](https://user-images.githubusercontent.com/37825063/38769446-b63d12e8-4046-11e8-917c-1add0c8af734.jpg)

**Comment by magamis** [comment]
![wowscrnshot_041518_005100](https://user-images.githubusercontent.com/37825063/38769504-86acc40a-4047-11e8-9bcc-a35261bc39f1.jpg)

**Comment by jeffreyhamby** [comment]
You can get the mobs down if you take A LOT of time, but they do evade quite a bit. Also, the dragons phase in and out and are impossible to kill, making the quest un-completeable.

**Comment by Golganeth** [comment]
Duplicate of #5386. If the issue persists, please give any extra feedback once the fix is applied on the live realm (when the tag gets updated).

Closing up
